Address.py

```diff
@@ -1,11 +1,10 @@
 class Address:
-    def __init__(self, address: str = "", apartment: str = "", city: str = "", state: str = "", country: str = "", zip: str = "") -> None:
-        self.address = address
-        self.apartment = apartment
-        self.city = city
-        self.state = state
-        self.country = country
-        self.zip = zip
+    def __init__(self, address: str = "", apartment: str = "", city: str = "", state: str = "", country: str = "", zip: str = "", **kwargs) -> None:
+        self.address = kwargs.get("Address", "")
```

Comment:
Address class is not actually needed, it can be removed and its members move to be part of employee directly.(optional,low priority)

Reply:
Yeah, I agree, I think we ended up doing it because the professor mentioned it. We can remove it at the end if we don't need it.